examples/pdf-bookmarks.cc

```diff
@@ -73,10 +73,11 @@ void show_bookmark_details(QPDFOutlineObjectHelper outline,
 
       case st_numbers:
         QTC::TC("examples", "pdf-bookmarks numbers");
-        for (std::vector<int>::iterator iter = numbers.begin();
-             iter != numbers.end(); ++iter)
+        // for (std::vector<int>::iterator iter = numbers.begin();
```

Comment:
Delete the commented out code

Reply:
Sorry. Definite case of attention deficit.